llvm/lib/Target/RISCV/MCTargetDesc/RISCVInstPrinter.h

```diff
@@ -31,7 +31,7 @@ class RISCVInstPrinter : public MCInstPrinter {
   void printRegName(raw_ostream &O, MCRegister Reg) override;
 
   void printOperand(const MCInst *MI, unsigned OpNo, const MCSubtargetInfo &STI,
-                    raw_ostream &O, const char *Modifier = nullptr);
+                    raw_ostream &O, StringRef Modifier = {});
```

Comment:
The parameter has been deprecated a long time ago. It should be removed from function declarations wherever possible. RISCV version is never called with the Modifier. I didn't check other targets, but I guess half of them don't need the parameter either.

Reply:
SG, let me remove them where possible